app/assets/stylesheets/about.css.scss

```diff
@@ -0,0 +1,10 @@
+#about > ul {
+  margin-left: 15px;
+}
+
+// Quick hack to override jquery-ui.custom.css
+span.ui-button-icon-primary.ui-icon.ui-icon-closethick {
```

Comment:
This is selector is really a quick hack, a better selector should be written when we have time.